src/elm/Layout.elm

```diff
@@ -120,6 +120,10 @@ withOnQueryParameterChanged { key, onChange } (Layout layout) =
     Layout { layout | onQueryParameterChangedDict = Dict.insert key onChange layout.onQueryParameterChangedDict }
 
 
+
+-- USED INTERNALLY BY ELM LAND
+
+
```

Comment:
i believe this will get overwritten by `elm-land generate`, should we remove it?

Reply:
Missed that one when removing comments from the top level files. I'll remove it.